File: leetcode/557. Reverse Words in a String 3/557_reverse_words_in_a_string_3.py
class Solution:
    def reverseWords(self, s: str) -> str:
        words = s.split()
        result = []
        for i in range(len(words)):
            result.append(words[i][::-1])
        
        output = ""
        for i in range(len(result)):
            output += result[i] + " "
        return output[:-1]

File: leetcode/557. Reverse Words in a String 3/557_reverse_words_in_a_string_3.cs
public class Solution {
    public string ReverseWords(string s) {
        string[] word_arr = s.Split(' ');
        string result = "";
        
        for(int i=0; i< word_arr.Length; i++)
        {
            word_arr[i] = ReverseWord(word_arr[i]);
        }
        result = string.Join(" ", word_arr);
        return result;
    }
    
    private string ReverseWord(string word)
    {        
        return new string(word.Reverse().ToArray());
    }
}
